### PR TITLE
implement submitForm function

### PR DESCRIPTION
**Pull request by Suuuuha**
related to #90

### CODE REVIEW

File: app/public/map.js [hunk 1]
```diff
@@ -194,6 +199,31 @@ var nazarethNuggets = (function () { // eslint-disable-line
   // on zoomend is good but not perfect, because can zoom multiple levels before this function will re-run
   mymap.on('zoomend', displayCorrectIcons)
 
+  function submitForm (e) { // eslint-disable-line
+    // get data from form fields
+    var form = e.target.parentNode.parentNode
+    var inputs = form.querySelectorAll('.add-form-input')
+    var formData = {}
+    inputs.forEach(function (input) {
+      formData[input.name] = input.value
```
Comment (Karyum) [inline app/public/map.js:208]: so im guessing we need to add a name to each input in the form??

**Review by Karyum — APPROVED** [review]

**Comment by Karyum** [comment]
isn't this pull request related to issue #90 ?? cause in the description it says #92

**Comment by mattlub** [comment]
yep updated @Karyum

File: app/public/map.js (hunk 1 above)
Comment (mattlub) [reply app/public/map.js:208]: yeh I guess... or we merge your form and then we add the name attributes to make this branch work

**Review by Karyum — APPROVED** [review]

**Review by HilbertSpitzer — APPROVED** [review]

**Pull request merged**